app/src/main/java/com/artemzin/qualitymatters/models/PicassoBitmapClient.java

```diff
@@ -0,0 +1,21 @@
+package com.artemzin.qualitymatters.models;
+
+import android.widget.ImageView;
+import com.squareup.picasso.Picasso;
+
+import javax.inject.Inject;
+
+public class PicassoBitmapClient implements NetworkBitmapClient {
+
+  private final Picasso picasso;
+
+  @Inject
+  public PicassoBitmapClient(final Picasso picasso) {
+    this.picasso = picasso;
+  }
+
+  @Override
+  public void downloadInto(String url, ImageView imageView) {
+    picasso.load(url).fit().centerCrop().into(imageView);
```

Comment:
Good mention, in project we can have more than one place where we need to use image library  with different setups and i think is better/easier to create implementation per use case rather than one generic implementation or to invent something like builder, builder wrapper for library properties ))